ansible/roles/jenkins-workspace/files/clean_binary_tmp.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -ex
+
+cd ~binary_tmp/binary_tmp.git
+(echo; date) >> ~binary_tmp/clean_binary_tmp.log
+
+git fetch origin +master:master
+
+for b in $(git branch | sed /\*/d); do
+  if [ -z "$(git log -1 --since='4 weeks ago' -s $b)" ]; then
```

Comment:
```suggested
  if [ -z "$(git log -1 --since='5 days ago' -s $b)" ]; then
```
IMO 5 days is enough. After five days we should run a rebuild not a resume (since `master` probably progressed).

Reply:
agreed, 4 weeks is a long time, any good reason for that @joaocgreis?

Reply:
I find that after a week the jobs are most often no longer in the history anyway.  Maybe 7 days?

Reply:
7 days sounds reasonable, updated.